libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -31,12 +33,6 @@ export class BookSearchComponent implements OnInit {
     return this.searchForm.value.term;
   }
 
-  ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
-      this.books = books;
-    });
-  }
-
   formatDate(date: void | string) {
```

Comment:
What about this function? Have you given any thought to whether this is appropriate or not?

Reply:
Used angular date pipe to achieve this format